01_prepare_host.sh

```diff
@@ -6,6 +6,10 @@ source lib/logging.sh
 # shellcheck disable=SC1091
 source lib/common.sh
 
+sudo modprobe -r -a kvm_intel kvm
+sudo modprobe kvm tdp_mmu=0
+sudo modprobe -a kvm kvm_intel
```

Comment:
TODO: revert this once the openstack image build fixed

Reply:
you are doing two things here. 
downgrading and removing modules. if you update the title, it would help us search in the future.